checkbox-ng/checkbox_ng/resume_menu.py

```diff
@@ -0,0 +1,343 @@
+# This file is part of Checkbox.
+#
+# Copyright 2023 Canonical Ltd.
+# Written by:
+#   Maciej Kisielewski <maciej.kisielewski@canonical.com>
+#
+# Checkbox is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3,
+# as published by the Free Software Foundation.
+#
+# Checkbox is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Checkbox.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+This module contains Uriwd components used on Resume Session screen.
+"""
+import urwid
+
+from collections import namedtuple
+
+palette = [
+    ("focus", "black", "light gray", "standout"),
+    ("body", "light gray", "black", "standout"),
+    ("head", "black", "light gray", "bold"),
+    ("buttnf", "black", "light gray"),
+    ("buttn", "light gray", "black", "bold"),
+    ("foot", "light gray", "black"),
+    ("start", "dark green,bold", "black"),
+]
+
+
+class PassiveListBox(urwid.ListBox):
+    """
+    A ListBox that is not selectable. It is used to display information
+    about the currently focused session.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._selectable = False
```

Comment:
We have to make sure the info panel is big enough to not need to access it and scroll through it.

Reply:
Moved back to columns in 5799e4d11d83b843cce725172d0dc0cbe0b65706